src/main/app/src/app/admin/parameter-edit/parameter-edit.component.ts

```diff
@@ -106,6 +106,7 @@ export class ParameterEditComponent
     super(utilService);
     this.route.data.subscribe((data) => {
       this.parameters = data.parameters;
+      console.log("IntakeMail: " + this.parameters.intakeMail);
```

Comment:
Is this intentional or left from debugging?

Reply:
ah good one. not intentional no. :-)

Reply:
removed it